.github/workflows/process-load-test-result.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           while [ ! -z "$status" ]
           do
             sleep 20
-            status=$(gh pr status --jq '.currentBranch .statusCheckRollup[] | select(.status!="COMPLETED") | select(.context!="license/cla")' --json statusCheckRollup)
+            status=$(gh pr status --jq '.currentBranch .statusCheckRollup[] | select(.status!="COMPLETED") | select(.context!="license/cla" | select(.context!="codecov/project" )' --json statusCheckRollup)
```

Comment:
This wont wait for the codecov status right?
This will just ignore the checks with context `codecov/project`

Reply:
Fixed in 2da6fad